website/docs/layouts/flex/partials/code/component-api.md

```diff
@@ -0,0 +1,78 @@
+    If an `@isInline` parameter is provided, then the element will be displayed as `inline-flex` (useful to achieve specific layouts). Otherwise, it will have a `flex` layout.
+  </C.Property>
+  <C.Property @name="...attributes">
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  </C.Property>
+</Doc::ComponentApi>
+
+### Contextual components
+
+#### [LF].Item
+
+The `Layout::Flex::Item` component, yielded as contextual component, to be used as child of the `flexbox` element to control its `basis/grow/shrink` values (and other properties).
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="tag" @type="string" @default="div">
+    HTML tag to be used to render the flex item element.
+  </C.Property>
+  <C.Property @name="basis" @type="string|0">
+    The value (size) of the CSS `flex-basis` property, which sets the initial main size of the flex item (for a technical explanation: [see MDN reference](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-basis)).
+    <br/><br/>
+    <em>Notice: when the value is set to `0`, a CSS class is used to apply the `flex-basis` property. In all other cases, an inline `style` declaration is used to accommodate all the possible values that this CSS property can have.</em>
+  </C.Property>
+  <C.Property @name="grow" @type="boolean|number|string">
+    The value (size or keyword) of the CSS `flex-grow` property, which sets the flex grow factor of the flex item (for a technical explanation:  [see MDN reference](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-grow)).
+    <br/><br/>
+    <em>Notice: when the value is set to `true/false` or `0/1` a CSS class is used to apply the `flex-grow` property. In all other cases, an inline `style` declaration is used, to accommodate all the possible values that this CSS property can have.</em>
+  </C.Property>
+  <C.Property @name="shrink" @type="boolean|number|string">
+    The value (size or keyword) of the CSS `flex-shrink` property, which sets sets the flex shrink factor of the flex item (for a technical explanation: [see MDN reference](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-shrink)).
+    <br/><br/>
+    <em>Notice: when the value is set to `true/false` or `0/1` a CSS class is used to apply the `flex-shrink` property. In all other cases, an inline `style` declaration is used, to accommodate all the possible values that this CSS property can have.</em>
+  </C.Property>
+  <C.Property @name="enableCollapseBelowContentSize" @type="boolean">
+    When this special argument is set to `true`, it applies `min-width: 0` to the element, allowing the flex item to shrink below its content's intrinsic minimum width (e.g., used with text truncated with ellipses)
+    <br/><br/>
+    <em>Notice: this may have accessibility implications; be considerate in how this special argument is used and how this impacts the content of the flex item element.</em>
```

Comment:
```suggestion
    <em>Note: this may have accessibility implications; be considerate in how this special argument is used and how this impacts the content of the flex item element.</em>
```

Suggestion: Could we be more explicit about what the issue is that they should look for?